### DISCUSSION TITLE
Emulator problems

### DISCUSSION THREAD

**Discussion by Uday-kiran9147** [adoontheway/better_flutter#3] [General]
While working on flutter applications everything works fine and in the middle of programming when I make changes, Sometimes My Emulator will not run the changes I have made, and then after It Completely stops Running flutter code, This makes me stop development. CAN ANYONE SOLVE my problem?

**Comment by adoontheway** [comment]
Is there any errors appeared in your debug console?
If not, you should take a video or something to show us.

**Reply by Uday-kiran9147** [reply]
I 'll do that. Thank you!

**Reply by adoontheway** [reply]
that is the last option and not the best choice. 
you should post the console log or emulator log first.

**Comment by adoontheway** [comment]
long time no updates.
